sheetstorm/management/commands/generate_test_data.py

```diff
@@ -170,6 +200,112 @@ def _create_start_date(time_delta: Any = PROJECT_START_DATE_TIME_DELTA) -> timez
     def _create_stop_date(time_delta: Any = PROJECT_STOP_DATE_TIME_DELTA) -> timezone.datetime:
         return timezone.now() - time_delta
 
+    def execute_adding_users_to_projects(self) -> None:
+        self.add_users_to_projects(
+            CustomUser.UserType.EMPLOYEE.name,
+            ProjectType.SUSPENDED.name,
+            self.max_number_of_employees_in_suspended_project,
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.EMPLOYEE.name, ProjectType.ACTIVE.name, self.max_number_of_employees_in_active_project
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.EMPLOYEE.name,
+            ProjectType.COMPLETED.name,
+            self.max_number_of_employees_in_completed_project,
+        )
+
+        self.add_users_to_projects(
+            CustomUser.UserType.ADMIN.name, ProjectType.SUSPENDED.name, self.max_number_of_admins_in_suspended_project
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.ADMIN.name, ProjectType.ACTIVE.name, self.max_number_of_admins_in_active_project
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.ADMIN.name, ProjectType.COMPLETED.name, self.max_number_of_admins_in_completed_project
+        )
+
+        self.add_users_to_projects(
+            CustomUser.UserType.MANAGER.name,
+            ProjectType.SUSPENDED.name,
+            self.max_number_of_managers_in_suspended_project,
+            as_manager=True,
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.MANAGER.name,
+            ProjectType.ACTIVE.name,
+            self.max_number_of_managers_in_active_project,
+            as_manager=True,
+        )
+        self.add_users_to_projects(
+            CustomUser.UserType.MANAGER.name,
+            ProjectType.COMPLETED.name,
+            self.max_number_of_managers_in_completed_project,
+            as_manager=True,
+        )
+
+        self.remove_all_random_user_projects(CustomUser.UserType.EMPLOYEE.name)
+        self.remove_all_random_user_projects(CustomUser.UserType.ADMIN.name)
+        self.remove_all_random_user_projects(CustomUser.UserType.MANAGER.name)
+
+    def add_users_to_projects(
+        self, user_type: str, project_type: str, max_number_of_users_in_projects: int, as_manager: bool = False
+    ) -> None:
+        users_list = self._get_list_of_users(user_type)
+        projects_list = self._get_list_of_projects(project_type)
+
+        for project in projects_list:
```

Comment:
`_get_list_of_users` function should be called once for every user type instead of calling it in every combination with projects. `_get_list_of_projects` analogically